test/result/dtype/blob.result

```diff
@@ -101,3 +101,13 @@ substring(a, 1, 4)    length(a)
 3456    6
 drop table t1;
 drop table t2;
+
+drop table if exists test;
+select hex( 0x01020304FFFFFFFF0000000CAACB0000);
+hex(0x01020304ffffffff0000000caacb0000)
+01020304ffffffff0000000caacb0000
+create table test(a blob);
+insert into test values(0x616162636465666768202122232425303132);
+select * from test;
+a
+aabcdefgh !"#$%012
```

Comment:
different from result of mysql

Reply:
yes, it's different, that's because the bvt's result will use the mysql-client's binary-revert, but in local machine, the result will be 0x616162636465666768202122232425303132